src/utils/resizeShadowCameraFrustum.js

```diff
@@ -53,7 +53,7 @@ export default function resizeShadowCameraFrustum(light, scene) {
   verts[6].set(max.x, max.y, min.z);
   verts[7].set(max.x, max.y, max.z);
 
-  inverseLightMatrixWorld.getInverse(light.shadow.camera.matrixWorld);
+  inverseLightMatrixWorld.getInverse(light.matrixWorld);
```

Comment:
You are reading an objects matrixWorld here (and above), so you should call `updateMatrices` (as per https://github.com/mozilla/hubs/blob/master/src/utils/threejs-world-update.js#L20). Since you are updating the matrices at the calling location it will be a noop (since the dirty flags wont be set). Alternatively we can note somehow that this function is pushing this responsibility to the caller (maybe we need a naming convention for this)